.github/workflows/pr_metadata.yml

```diff
@@ -0,0 +1,23 @@
+# Auto-update metadata on pull requests
```

Comment:
It's helpful to have a link here to the "schema":
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions

Reply:
Done in 1b51ab0